shared/team-building/service-tab-bar.desktop.tsx

```diff
@@ -4,76 +4,156 @@ import * as Styles from '../styles'
+                type="icon-progress-grey-animated"
+                color={Styles.globalColors.greyDark}
+                style={styles.pendingIcon}
+              />
+            ))}
+        </Kb.Box2>
+      </Kb.Box2>
+      <Kb.Box2
+        direction="horizontal"
+        fullWidth={true}
+        style={Styles.collapseStyles([
+          props.isActive ? styles.activeTabBar : styles.inactiveTabBar,
+          props.isActive && {backgroundColor: serviceIdToAccentColor(props.service)},
+        ])}
+      />
+    </Kb.ClickableBox>
+  )
+}
+
+const MoreNetworksButton = Kb.OverlayParentHOC(
+  (
+    props: Kb.PropsWithOverlay<{
+      services: Array<ServiceIdWithContact>
+      onChangeService: (service: ServiceIdWithContact) => void
+    }>
+  ) => (
+    <>
+      <Kb.Box2
+        direction="vertical"
+        style={styles.moreNetworks1}
+        fullHeight={true}
+        centerChildren={true}
+        ref={props.setAttachmentRef}
+      >
+        <Kb.WithTooltip text="More networks" containerStyle={styles.moreNetworks2}>
+          <Kb.ClickableBox onClick={props.toggleShowingMenu} style={styles.moreNetworks3}>
+            <Kb.Text type="BodyBigExtrabold" style={styles.moreText}>
+              •••
+            </Kb.Text>
+          </Kb.ClickableBox>
+        </Kb.WithTooltip>
+      </Kb.Box2>
+      <Kb.FloatingMenu
+        attachTo={props.getAttachmentRef}
+        closeOnSelect={true}
+        items={props.services.map(service => ({
+          onClick: () => props.onChangeService(service),
+          title: service,
+          view: <MoreNetworkItem service={service} />,
+        }))}
+        onHidden={props.toggleShowingMenu}
+        visible={props.showingMenu}
+      />
+    </>
+  )
+)
+
+const MoreNetworkItem = (props: {service: ServiceIdWithContact}) => (
+  <Kb.Box2 direction="horizontal" fullHeight={true} alignItems="center">
+    <Kb.Icon
+      style={{marginRight: Styles.globalMargins.tiny}}
+      fontSize={16}
+      type={serviceIdToIconFont(props.service)}
     />
-  </Kb.ClickableBox>
+    <Kb.Text type="Body">{serviceIdToLongLabel(props.service)}</Kb.Text>
+  </Kb.Box2>
 )
 
 const undefToNull = (n: number | undefined | null): number | null => (n === undefined ? null : n)
 
-export const ServiceTabBar = (props: Props) => (
-  <Kb.Box2 direction="horizontal" fullWidth={true} style={styles.tabBarContainer}>
-    {Constants.services.map(service => (
-      <ServiceIcon
-        key={service}
-        service={service}
-        label={serviceIdToLongLabel(service)}
-        labelPresence={1}
-        onClick={() => props.onChangeService(service)}
-        count={undefToNull(props.serviceResultCount[service])}
-        showCount={props.showServiceResultCount}
-        isActive={props.selectedService === service}
-      />
-    ))}
-  </Kb.Box2>
-)
+export const ServiceTabBar = (props: Props) => {
+  const [lastSelectedUnlockedService, setState] = React.useState<ServiceIdWithContact | null>(null)
```

Comment:
```suggestion
  const [lastSelectedUnlockedService, setSelectedUnlockedService] = React.useState<ServiceIdWithContact | null>(null)
```

think this is a little more descriptive as to what state you're setting 🙂